gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-info.spec.ts

```diff
@@ -0,0 +1,121 @@
+    cy.getByDataTestId('api_info_versionfield').should('be.visible');
+    cy.getByDataTestId('api_info_descriptionfield').should('be.visible');
+    cy.getByDataTestId('api_info_labelsfield').should('be.visible');
+    cy.getByDataTestId('api_info_categoriesdropdown').should('be.visible').click();
+    cy.getByDataTestId('api_info_categorieslist-unavailable').should('be.visible');
+  });
+
+  it('Edit Info Page and Verify changes', () => {
+    cy.getByDataTestId('api_list_edit_button').first().click();
+    ApiDetails.info().click();
+    cy.getByDataTestId('api_info_namefield').scrollIntoView().clear().type(`${apiName}`);
+    cy.getByDataTestId('api_info_versionfield').clear().type(`${apiVersion}`);
+    cy.getByDataTestId('api_info_descriptionfield').clear().type(`${apiDescription}`);
+    cy.getByDataTestId('api_info_savebar', { timeout: 5000 }).contains('Save').click();
+    cy.contains('Configuration successfully saved!').should('be.visible');
+    cy.visit('/#!/environments/default/apis/');
+    cy.contains(`${apiName}`).should('be.visible');
+  });
+
+  it('Export API and verify json download', () => {
+    cy.getByDataTestId('api_list_edit_button').first().click();
+    cy.getByDataTestId('api_info_export_menu').click();
+    cy.getByDataTestId('api_info_export_api').click();
+    const downloadsFolder = Cypress.config('downloadsFolder');
+    cy.readFile(path.join(downloadsFolder, `${newName}-${apiVersion}.json`)).should('exist');
+  });
+
+  it('Duplicate API and verify duplicate', () => {
+    cy.getByDataTestId('api_list_edit_button').first().click();
+    cy.getByDataTestId('api_info_duplicate_menu').click();
+    cy.getByDataTestId('api_info_duplicate_path').type(`${newName}-duplicate`);
+    cy.getByDataTestId('api_info_duplicate_version').type(`${apiVersion}`);
+    cy.intercept('POST', '**/duplicate').as('duplicateApi');
+    cy.getByDataTestId('api_info_duplicate_api').click();
+    cy.wait('@duplicateApi').then((interception) => {
+      expect(interception.response.statusCode).to.eq(200);
+      expect(interception.response.body).to.have.property('id');
+      expect(interception.response.body).to.have.property('name', apiName);
+      expect(interception.response.body).to.have.property('version', `${apiVersion}`);
+      apiId = interception.response.body;
+      cy.contains('API duplicated successfully');
+    });
+    cy.contains('API duplicated successfully').should('be.visible');
+    cy.visit('/#!/environments/default/apis/');
+    cy.contains(`${newName}-duplicate`).should('be.visible');
+  });
+
+  it('Verify Promote pop-up on info page', () => {
+    cy.getByDataTestId('api_list_edit_button').first().click();
+    cy.getByDataTestId('api_info_promote').click();
+    cy.contains('Meet Cockpit').should('be.visible');
+    cy.getByDataTestId('api_info_promote_ok').click();
+  });
+
+  // Danger Zone buttons to be continued at a later stage. See JIRA (2075)
```

Comment:
```suggestion
  // TODO: Danger Zone buttons to be continued at a later stage. See JIRA (2075)
```

Or, if there is already a JIRA, I don't think you need this comment

Reply:
Okay no problem - removed comment.